source/_patterns/02-molecules/card/_card.scss

```diff
@@ -2,7 +2,8 @@ $_config_schemes: (
   base: (
     border-width: "05",
     hover-border-width: "05",
-    border-radius-container: "none", // Should be set by $theme-card-border-radius
+    border-radius-container: "none",
+    // Should be set by $theme-card-border-radius
```

Comment:
If this line break was done by prettier, then prob should move the comment up a line so it makes more sense